google/cloud/storage/client_object_acl_test.cc

```diff
@@ -71,12 +71,12 @@ TEST_F(ObjectAccessControlsTest, ListObjectAcl) {
 
   EXPECT_CALL(*mock, ListObjectAcl(_))
       .WillOnce(Return(
-          std::make_pair(TransientError(), internal::ListObjectAclResponse{})))
+          StatusOr<internal::ListObjectAclResponse>(TransientError())))
       .WillOnce(Invoke([&expected](internal::ListObjectAclRequest const& r) {
         EXPECT_EQ("test-bucket", r.bucket_name());
         EXPECT_EQ("test-object", r.object_name());
 
-        return std::make_pair(Status(),
+        return make_status_or(
```

Comment:
Join lines?

Reply:
Fixed (here and elsewhere). I guess I trusted clang-format too much.